src/CollAction/Views/Projects/Find.cshtml

```diff
@@ -9,118 +9,7 @@
     var defaultDisplayModel = FindProjectViewModel.DefaultDisplayProjectViewModel;
 }
 
-<h2>@Localizer["Find"]</h2>
```

Comment:
Also would you mind if we migrated the install steps to the README of the application. Might help new users finding us through git who don;t have access to atlassian cloud.